test/math.ts

```diff
@@ -53,4 +53,103 @@ test("Should be able to use the cos function with degrees and radians", (assert)
     ~~~
   `);
   assert.end();
-})
\ No newline at end of file
+})
+
+test("Test that string concatenation is still working after NaN change.", (assert) => {
+  let expected = {
+    insert: [
+      ["a", "tag", "div"],
+      ["a", "text", "Test Testy"],
+    ],
+    remove: [],
+  };
+
+  evaluate(assert, expected, `
+    ~~~
+    search
+      a = "Test "
+      b = "Testy"
+      x =  a + b
+    bind @browser
+    [#div text: x]
+    ~~~
+  `);
+  assert.end();
+});
+
+test("Divide by zero should return nothing.", (assert) => {
+  let expected = {
+    insert: [],
+    remove: [],
+  };
```

Comment:
I would suggest adding `errors: true` here to make sure that we effectively have an error.